packages/datadog-instrumentations/src/querystring.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const { addHook, channel } = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+const names = ['querystring', 'node:querystring']
+
+const querystringParseCh = channel('datadog:querystring:parse:finish')
```

Comment:
why do we need this? isn't this redundant?
We have instrumentation for req.query getter in express, why do we need to instrument this?

Reply:
I was thinking maybe for IAST, but if you're also confused then idk hahaha

Reply:
This [test](https://github.com/DataDog/system-tests/blob/56b1a4753ecc30b8f93373f9a4e57f05a259b69d/tests/appsec/iast/source/test_parameter_value.py#L44) fails because in express v4 queries are read directly via [query middleware](https://github.com/expressjs/express/blob/4.21.1/lib/middleware/query.js#L27), and IAST subscribes to this event [here](https://github.com/DataDog/dd-trace-js/blob/9d0f9fa7e53c56a3e01d5524e7dd7b61354ca1cd/packages/dd-trace/src/appsec/iast/taint-tracking/plugin.js#L49).

In express v5, query parsing uses both `qs` (same as express v4) and `querystring` ([code here](https://github.com/expressjs/express/blob/9e3dbb43744620b8b490c61c9269a168e0c0e3ae/lib/utils.js#L142)), so we need to instrument `querystring` to ensure the same functionality.

Reply:
@iunanua I removed `qs` and `querystring` instrumentation so we do rely only on express